device-types/Brocade/ICX-6450-48P.yaml

```diff
@@ -0,0 +1,128 @@
+---
+manufacturer: Brocade
+model: ICX 6450-48P
+slug: brocade-icx6450-48p
+part_number: ICX6450-48P
+is_full_depth: false
+u_height: 1
+airflow: side-to-rear
+weight: 14.11
+weight_unit: lb
+interfaces:
+  - name: GigEthernetmgmt1
+    type: 1000base-t
+    mgmt_only: true
+  - name: GigabitEthernet1/1
+    type: 1000base-t
```

Comment:
Looking at the spec sheet these are POE+ ports therefore all POE+ ports need the following:
`poe_mode: pse`
`poe_type: type2-ieee802.3at`